src/calendar/tests/CalendarMonth.spec.js

```diff
@@ -35,6 +35,7 @@ describe('The CalendarMonth Component', function () {
         onMonthChange: function () {},
         onYearChange: function () {},
         bemBlock: 'DateRangePicker',
+        locale: 'en',
```

Comment:
@thaerlabs Would you mind adding this default functionality as part of this PR?

It means that this PR could go out without breaking API backwards compatability